### DISCUSSION TITLE
希望增加动态中课程屏蔽功能，以及左上角访问’搜索‘的快捷方式

### DISCUSSION THREAD

**Discussion by Oldnopower** [the1812/Bilibili-Evolved#4692] [功能建议 / Ideas]
开发者您好，

谢谢您制作的软件，我特别喜欢这些功能！
但是在使用过程中，我有一些个人希望增加的功能，希望您可以考虑加入。

**1.希望增加动态中课程屏蔽** 
如下图所示，这是动态页面，取消关注up主即不会出现其动态，但是付费课程部分目前并没有取消或者屏蔽的方法，已经向b站反映过，但是并没有收到明确答复，所以希望加入这个功能。
![P3WP{(IFM4ZC7C4R{QOT 5W](https://github.com/the1812/Bilibili-Evolved/assets/60099789/80a1a343-c8a0-4937-8ba2-c2261befb91b)

**2.左上角访问’搜索‘页面的快捷方式**
如下图图一所示，b站单独的搜索页面十分干净，我更倾向于使用这个页面作为主页，可是在左上角的链接中，并没有单独的‘搜索’链接（见图二），所以希望可以在左上角添加一个访问搜索页面的链接以便于使用。
![U795 EY0QN09@DE) DTV4KQ](https://github.com/the1812/Bilibili-Evolved/assets/60099789/d9b7c0f1-8a3b-40fd-8e45-7f55905c2df4)
![( $6 Q1ZNZ}S6WJ)3YFBTZB](https://github.com/the1812/Bilibili-Evolved/assets/60099789/37ca9dea-cbfa-4744-8abb-a1374bf070d1)

十分感谢！

**Comment by Geeyun-JY3** [comment]
2. 你说的左上角是希望原版顶栏有个这样的入口吗？其实你可以试试`自定义顶栏`的搜索框，未输入任何关键词时点击最右 `>` 执行按钮即会打开你要的搜索首页（你附的图二看着就是本脚本的`自定义顶栏`啊）。

**Reply by Geeyun-JY3** [reply]
2. 如果你是希望原版顶栏有个这样的入口，#801 的第 1 点实现后点击原版顶栏的搜索按钮也能打开你要的搜索首页，只是 4 年看下来要实现应该还是遥遥无期。

**Reply by Oldnopower** [reply]
十分感谢！这个办法很好！
